tests/e2e/pageObjects/components/common/button.ts

```diff
@@ -0,0 +1,6 @@
+import { Selector } from 'testcafe';
+
+export class Button {
```

Comment:
can we raname the class
since button for me can't contain list of locators ( it can take locator and has some actions like click hover) but if we just store only 2 locators we need to specify what exact locators we have

Reply:
renamed to EditorButton

Reply:
actually I created this class more for future, to move all common locators related to buttons there